package.json

```diff
@@ -18,21 +18,20 @@
   ],
   "type": "module",
   "engines": {
-    "node": ">=20.11"
+    "node": ">=20.12"
```

Comment:
the luke document has to be updated to specify node>=20.12 to make this work, right?

Reply:
It probably should be updated at some point, but just updating this package will be enough for `npm` to complain about the engine version, if you're running < 20.12.